docker-compose.yml

```diff
@@ -1,50 +1,42 @@
-version: '3'
+version: '3.9'
 
 services:
     
   db:
     image: postgres
     volumes:
       - ./tmp/db:/var/lib/postgresql/data
+      - ./db/init.sql:/docker-entrypoint-initdb.d/init.sql
     restart: always
-    env_file:
-      - .env
     environment:
-      POSTGRES_USER: ${DB_USERNAME}
-      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
```

Comment:
Since the docker environment created with `docker-compose` is always a test environment, it's okay to hard code the environment variables here (in fact I think this is the convention. Almost all examples I come across does it like this 🤔)